src/ExadelMentorship.WebApi/Controllers/WeatherController.cs

```diff
@@ -45,5 +48,11 @@ public Task<IEnumerable<City>> GetFutureWeather([FromRoute] string cityName, [Fr
                 }
             );
         }
+
+        public string EnableJobs()
+        {
+            _weatherJob.HistorySaving();
```

Comment:
This probably should trigger during app start and most likely be idempotent

Reply:
done